### ISSUE TITLE
weights of the model posted in the paper?

### ISSUE THREAD

**Issue by krishnaadithya** [Retinal-Research/NN-MOBILENET#4]
Hello

I am unable to find the DR model weights in the github repo.

**Comment by ChongQingNoSubway** [comment]
We are not considering releasing weights due to the extensive number involved in evaluating various datasets through cross-validation. The datasets mentioned in the manuscript are open-source and relatively small; you could train from scratch(we have provided the pre-trained weight). We employed Dropout for robustness, and the reproduced results are expected to closely align with those presented in the paper.
